services/web/server/tests/unit/with_dbs/docker-compose-devel.yml

```
@@ -62,7 +62,14 @@ services:
     image: rediscommander/redis-commander:latest
     restart: always
     environment:
-      - REDIS_HOSTS=resources:redis:6379:0,locks:redis:6379:1,validation_codes:redis:6379:2,scheduled_maintenance:redis:6379:3,user_notifications:redis:6379:4,announcements:redis:6379:5
+      - >-
+        REDIS_HOSTS=
+        resources:redis:6379:0:${TEST_REDIS_PASSWORD},
+        locks:redis:6379:1:${TEST_REDIS_PASSWORD},
+        validation_codes:redis:6379:2:${TEST_REDIS_PASSWORD},
+        scheduled_maintenance:redis:6379:3:${TEST_REDIS_PASSWORD},
+        user_notifications:redis:6379:4:${TEST_REDIS_PASSWORD},
+        announcements:redis:6379:5:${TEST_REDIS_PASSWORD}
```

Please add these dbs them here as well
```suggestion
        announcements:redis:6379:5:${TEST_REDIS_PASSWORD},
        distributed_identifiers:${REDIS_HOST}:${REDIS_PORT}:6:${REDIS_PASSWORD},
        deferred_tasks:${REDIS_HOST}:${REDIS_PORT}:7:${REDIS_PASSWORD}
```